src/store/modules/announcements.js

```diff
@@ -0,0 +1,52 @@
+import api from '../../api/api';
+
+export default {
+  namespaced: true,
+  state: {
+    announcements: [],
```

Comment:
I'm confused, I thought you said this had been separated out? https://c4cneuteams.slack.com/archives/C016RC79ZD2/p1598574066015400?thread_ts=1598568973.013300&cid=C016RC79ZD2